config/lib/app.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var config = require('../config'),
+	mongoose = require('./mongoose'),
+	express = require('./express'),
+	chalk = require('chalk');
+
+// Initialize Models
+mongoose.loadModels();
```

Comment:
Makes sense. I don't think it's terribly redundant. You definitely want to provide an ability to load them again, for whatever reason. I'm interested in how it could be implemented to *not* load the models when requiring the app. But that's not the scope of what you're doing here; you've made it very flexible. Carry on :)